client/my-sites/stats/features/modules/stats-utm/stats-module-utm.jsx

```diff
@@ -291,11 +261,6 @@ const StatsModuleUTM = ( {
 							</div>
 						}
 					/>
-					{ showFooterWithDownloads && (
-						<div className="stats-module__footer-actions stats-module__footer-actions--summary">
-							<UTMExportButton data={ data } fileName={ fileNameForExport } />
-						</div>
-					) }
```

Comment:
I see two instances of the button being removed but only one instance of it being replaced. Do we need the `downloadCsv` prop to be included here as well?

It looks like this 2nd code path is maybe dead now that the related feature flag has shipped. Maybe we follow up with a PR to remove code related to the "stats/empty-module-traffic" feature flag.

Reply:
Thanks for noting it! I just added the export button on the legacy path for consistency and created a [task](https://github.com/Automattic/wp-calypso/issues/101517) on the maintenance board to cleanup the `stats/empty-module-traffic` feature flag.